<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalarBarWidget</class>
 <widget class="QWidget" name="ScalarBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QCheckBox" name="displayCheckBox">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="titleGroupBox">
     <property name="title">
      <string>Title</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="titleFontLabel">
        <property name="text">
         <string>Font Setting:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="titleFontButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>titleEdit</zorder>
     <zorder>titleLabel</zorder>
     <zorder>titleFontLabel</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Label Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numberLabel">
        <property name="text">
         <string>Number of labels :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSpinBox" name="numberSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="formatLabel">
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TooltipLabel" name="formatHelpLabel">
          <property name="toolTip">
           <string>Specify the number format in printf format. For example:
   &quot;%5.2f&quot; for 3.141592 --&gt; 3.14
   &quot;%4.2e&quot; for 12300  --&gt; 1.23e+004</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="labelFormatEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="orientationLabel">
        <property name="text">
         <string>Orientation :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="verticalRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Vertical</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="horizontalRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelFontLabel">
        <property name="text">
         <string>Font Setting:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="labelFontButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="sizeGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Size</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="sizeLabel">
          <property name="text">
           <string>Specify Color bar size as rate against window size.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="widthLabel">
            <property name="text">
             <string>Width :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TooltipLabel" name="sizeWidthLabel">
            <property name="toolTip">
             <string>If you specify 1, Color bar width is the same to window width.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="widthSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="HeightLabel">
            <property name="text">
             <string>Height :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TooltipLabel" name="sizeHeightLabel">
            <property name="toolTip">
             <string>If you specify 1, Color bar height is the same to window height.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="heightSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="positionGroupBox">
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>Specify Color bar left bottom position as rate against window size.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="xLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>X :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TooltipLabel" name="positionXLabel">
            <property name="toolTip">
             <string>If you specify 0, Color bar left edge is located at window left edge.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="positionXSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yLabel">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Y :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TooltipLabel" name="positionYLabel">
            <property name="toolTip">
             <string>If you specify 0, Color bar bottom is located at window bottom.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="positionYSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TooltipLabel</class>
   <extends>QLabel</extends>
   <header>guibase/tooltiplabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="guibase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verticalRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>heightSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>positionXSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>positionYSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>titleEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>titleFontButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelFontButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelFormatEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
